lighthouse-core/report/html/templates.html

```diff
@@ -574,6 +574,218 @@
   </a>
 </template>
 
+<!-- Lighthouse PWA badge gauge -->
+<template id="tmpl-lh-gauge--pwa">
+  <style>
+    .lh-gauge--pwa {
+      width: calc(3 * var(--header-font-size));
```

Comment:
is there significance to this `3` or just feels about right?

Reply:
> is there significance to this 3 or just feels about right?

I copied this over from `.lh-gauge` since `lh-gauge` (and its modifiers) had a bunch of svg-score-gauge-specific properties.  Basically it just means `60px` :)

This is one of the properties that make it seem we need a more general "this is a round-ish thing representing the overall category score" name that includes both the gauge we've had and the badge container (though we could just stick with `lh-gauge` and move the current gauge stuff down inside).

Reply:
lh-gauge defines `--circle-size: calc(3 * var(--header-font-size));`  in its wrapper. 

seems good to either do that here too or to move the `circle-size` into the `.lh-vars` block.

Reply:
> seems good to either do that here too or to move the `circle-size` into the `.lh-vars` block.

after discussion, moved into `.lh-vars` block. It makes sense that the gauges should stay the same size and it turns out we were using the same number in a few other places that could benefit from a single definition